### ISSUE TITLE
All extensions show "Reload Required"

### ISSUE THREAD

**Issue by pdanpdan** [microsoft/vscode#144900]
Issue Type: <b>Bug</b>

Open Extensions pane - all enabled extensions show the `Reload Required` button.

VS Code version: Code - Insiders 1.66.0-insider (6b7453421201c888b81eae5c04ec822a6423608f, 2022-03-11T06:27:47.212Z)
OS version: Linux x64 5.16.11-76051611-generic
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz (8 x 2900)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 1|
|Memory (System)|31.08GB (0.85GB free)|
|Process Argv|--unity-launch --crash-reporter-id 609a073c-e838-44e0-83cd-756d613399cf|
|Screen Reader|yes|
|VM|0%|
|DESKTOP_SESSION|pop-wayland|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|pop-wayland|
|XDG_SESSION_TYPE|wayland|
</details><details><summary>Extensions (32)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.2.4
project-manager|ale|12.5.0
color-info|bie|0.7.0
transformer|dak|1.12.0
vscode-eslint|dba|2.2.3
xml|Dot|2.5.1
gitlens|eam|12.0.4
vscode-html-css|ecm|1.11.0
vscode-npm-script|eg2|0.3.24
rest-client|hum|0.24.6
l13-diff|L13|1.0.3
i18n-ally|lok|2.8.1
rainbow-csv|mec|2.0.0
git-graph|mhu|1.30.0
typescript-javascript-grammar|ms-|0.0.55
vscode-typescript-tslint-plugin|ms-|1.3.3
sqltools|mtx|0.23.0
sqltools-driver-mssql|mtx|0.2.0
sqltools-driver-pg|mtx|0.2.0
sqltools-driver-sqlite|mtx|0.2.0
vetur|oct|0.35.0
vuehelper|oys|0.1.0
vscode-versionlens|pfl|1.0.9
material-icon-theme|PKi|4.14.1
vscode-sort-json|ric|1.20.0
vscode-gtm|s3r|0.2.0
todotasks|san|0.5.0
sass-indented|syl|1.8.19
language-stylus|sys|1.15.0
gitflow|vec|1.2.1
markdown-all-in-one|yzh|3.4.0
cursorCharCode|zei|0.2.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30422396
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805:30301674
pythonvspyt200:30323110
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30404738
wslgetstarted:30449409
vsclayoutctrt:30405800
dsvsc009:30440021
pythonvspyt640:30438691
vscscmwlcmc:30428972
dockersurvey3cf:30438169
pynewfile477:30451556

```

</details>

**Comment by tjx666** [comment]
I occur the same issue. Finally, I reinstall the insider version to fix this.

**Comment by misolori** [comment]
Can confirm as well, we're looking into it...

![image](https://user-images.githubusercontent.com/35271042/157900333-357dd1de-a97c-4d06-a3dc-4e29595897e8.png)

**Comment by alexdima** [comment]
After a bisect, it looks like dffa71af994ce24e6f3fa8c440f44afc961c5dc2 is the first bad commit
```
commit dffa71af994ce24e6f3fa8c440f44afc961c5dc2
Author: Sandeep Somavarapu <sasomava@microsoft.com>
Date:   Wed Mar 9 22:18:16 2022 +0100

    show reload button when extension with different targetplatform is updated

 src/vs/workbench/contrib/extensions/browser/extensionsActions.ts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)
```

**Comment by alexdima** [comment]
FYI @sandy081 I've pushed a revert via https://github.com/microsoft/vscode/commit/dfd530095900d98bdeeaa5cde514e00b75b8177e

**Comment by sandy081** [comment]
Thanks @alexdima for looking into it and reverting the change that caused it.

Will revert the revert and push real fix for this.

**Comment by sandy081** [comment]
Fixed by 4cc9f9eee2e7c77ac06f2d8e6c926a4e0523f398
